packages/brand-positioning-block/src/types.ts

```diff
@@ -1,3 +1,37 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+import { DraggableAttributes, DraggableSyntheticListeners } from '@dnd-kit/core';
+
 export type BlockSettings = unknown;
+
+export interface BoardProps {
+    test?: string;
+}
+
+export interface AxisProps {
+    test?: string;
+}
+
+export interface ItemProps {
+    xPosition: number;
+    yPosition: number;
+    src: string;
+    id: number;
+}
+
+export type ToolbarProps = {
+    draggableAttributes: DraggableAttributes;
+    draggableListeners: DraggableSyntheticListeners;
+    isDragging: boolean;
+};
+
+export interface BlockInjectButtonProps {
```

Comment:
Please use `type` everywhere, we don't use `interface`

Reply:
fixed